src/Illuminate/Collections/Traits/EnumeratesValues.php

```diff
@@ -1014,30 +1010,6 @@ public function __get($key)
         return new HigherOrderCollectionProxy($this, $key);
     }
 
-    /**
-     * Results array of items from Collection or Arrayable.
-     *
-     * @param  mixed  $items
-     * @return array<TKey, TValue>
-     */
-    protected function getArrayableItems($items)
```

Comment:
We may want to leave this in for compatibility purposes and just proxy it to the new `Arr::from()` method, then add a `@deprecated` annotation. 

If someone extended a `Collection` class and calls this method, it will no longer exist, throwing an exception.

```php
/**
 * Results array of items from Collection or Arrayable.
 * 
 * @deprecated 
 *
 * @param  mixed  $items
 * @return array<TKey, TValue>
 */
protected function getArrayableItems($items)
{
    return Arr::from($items);
}
```

Reply:
@stevebauman thank you, it makes sense👌

Reply:
@stevebauman On second thought, I decided to undeprecate `getArrayableItems` and use it everywhere in collections as it was before. This is great extension point if someone want to create collections from custom classes.

Reply:
Sounds good to me! 👍